docs/validator/economics.md

```diff
@@ -30,11 +30,11 @@ You can obtain a validator *seat* by:
 
 ### Stake your NEAR tokens
 
-Use [NEAR Shell](../development/near-clitool.md) to append your staking transaction to the current `proposals`. At the end of the `epoch`, if your proposal is sufficient, your node will become a validator - and will begin to generate rewards after one more epoch. If needed, you can increase anytime your `proposal` by signing a larger staking transaction using the same *stakingKey*.
+Use [NEAR Shell](../development/near-clitool.md) to generate your staking transaction and issue your `proposal` to become validator. At the end of the `epoch`, if your proposal is large enough, your node will become a validator - and will begin to generate rewards after one more epoch. If needed, you can increase anytime your `proposal` by signing a larger staking transaction using the same *stakingKey*.
 
-### Stake delegated NEAR tokens
+### Stake third-party NEAR tokens via delegation
```

Comment:
Should we mention here that delegation is custodial?

Reply:
Is this a trap? 😆
Delegation via smart contracts shouldn't need to be custodial - a delegator loads his tokens into it and begins to compound rewards (minus fees for the validator). It may become custodial only if the validator must know delegators' private keys.
From what I can see, it can be custodial only if the wallet is hosted within "some service" that won't give you the private keys of your wallet - and it's also being a validator.